.github/workflows/move-work-in-progress-prs.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Move PR To Work In Progress
+on:
+  issue_comment:
+    types: [created]
+    branch:
+      - main
+
+jobs:
+  move-pr-to-wip:
+    if: contains(github.event.comment.body, '/pr mark wip') && github.event.issue.pull_request

```

Comment:
Suggestion: I'm not sure if we get short-circuiting here, but it seems like a good idea in any event to phrase these conditions in order of increasing expense. For example, checking whether the "issue" is specifically a PR should be cheaper than doing a substring match, and it reads naturally.